src/KeyControls.ts

```diff
@@ -0,0 +1,59 @@
+class KeyControls {
+  keys: {};
+  
+  constructor() {
+    this.keys = {};
+
+    // Bind event handlers
+    document.addEventListener("keydown", e => {
+      if ([37, 38, 39, 40, 32].indexOf(e.which) >= 0) {
```

Comment:
What are these numbers? Woud it not mean that it's impossible to have KeyCotrnols work with different keys? What if we have 2 players?